src/main/java/com/cburch/logisim/fpga/hdlgenerator/HdlTypes.java

```diff
@@ -88,10 +88,10 @@ public String getTypeDefinition() {
       final var contents = new StringBuilder();
       if (Hdl.isVhdl()) {
         contents.append(
-            LineBuffer.formatVhdl("{{type}} {{1}} {{is}} {{array}} ( {{2}} {{downto}} 0 ) {{of}}std_logic_vector( ", 
+            LineBuffer.formatVhdl("{{type}} {{1}} {{is}} {{array}} ( {{2}} {{downto}} 0 ) {{of}} std_logic_vector( ", 
                 myTypeName, myNrOfEntries))
             .append(myGenericBitWidth == null ? Integer.toString(myBitWidth - 1) : String.format("%s - 1", myGenericBitWidth))
-            .append(LineBuffer.formatVhdl("{{downto}} 0);"));
+            .append(LineBuffer.formatVhdl(" {{downto}} 0);"));
```

Comment:
That's pretty fragile. Can you please at least add a comment to pay attention to leading space? OR maybe write like this:

    .append(LineBuffer.formatVhdl(" " + "{{downto}} 0);"));